pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -158,6 +158,7 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo) (*apps.Dep
 		{Name: "ROOK_OSD_UUID", Value: osd.UUID},
 		{Name: "ROOK_OSD_ID", Value: osdID},
 		{Name: "ROOK_OSD_STORE_TYPE", Value: storeType},
+		{Name: "CEPH_CONF", Value: osd.Config},
```

Comment:
```suggestion
		{Name: "CEPH_ARGS", Value: "-m $(ROOK_CEPH_MON_HOST)"},
```

Alternatively, you could have used this, so when we get rid of the `ceph.conf`, nothing will break.
Just make sure `config.StoredMonHostEnvVars()` is passed too.

Reply:
In the interest of future changes, can we use what Seb suggested (`CEPH_ARGS`)? Eventually the `ceph.conf` will not exist, and I'd like to not forget how to do this when I make those changes. :)